frontend/src/text_resources.js

```diff
@@ -421,7 +421,35 @@ let LOCALIZE = new LocalizedStrings({
             "If you are certain that you want to quit this session, click the “Quit test” button. You will be exited out of this test session and provided instructions to complete your withdrawal.",
           descriptionPart3: "Are you sure you want to quit this test?"
         }
-      }
+      },
+      emailList: [
```

Comment:
These should get added to `sampleEmibJson.js` because you'd get this list in your API response.

Reply:
Then it could get loaded into redux.

Reply:
I think your json response would just return an array of objects `addressBook` (for example `{name: "Caley", description: "Developer"}`) since that's the actual content you'd be asking the server for. That can be loaded into redux as `addressBook`. When you load it into your component, you could transform it into an array of objects (for example `{text: "Caley, Developer", value: "caley"}`), and add the `{ text: "None", value: "null" }` object since the objects are for display purposes and should be kept in the display. Each of these transforms can use a helper function that can also easily be tested!

Reply:
This is also something you can make use of defining a shape for in constants.js.

Reply:
resolved in 639faf0....63b7584